docs/api/IoT.js-API-TLS.md

```diff
@@ -57,6 +57,25 @@ var socket = tls.connect(opts, function() {
 });
 ```
 
+### tls.connect(port[,host][,options][,callback])
+- `port` {number} Port the client should connect to.
```

Comment:
The node.js doc lists 4 variants for connect. I think we should list our supported variants as well.